keepassxc-browser/_locales/en/messages.json

```diff
@@ -707,6 +707,18 @@
         "message": "Passwords are generated by KeePassXC using your password generation profile.",
         "description": "Password Generator option help text, second part."
     },
+    "optionsPredefinedSiteList": {
+        "message": "Predefined sites",
+        "description": "Predefined site list details button text."
+    },
+    "optionsUsePredefinedSites": {
+        "message": "Use predefined sites for compatibility.",
+        "description": "Use predefined sites checkbox text."
+    },
+    "optionsUsePredefinedSitesHelpText": {
+        "message": "Adds most common sites automatically to Site Preferences and enables the Username-only option. Click below to see the site list.",
```

Comment:
Recommend: "Adds common sites that have username-only login forms to the site preferences"